include/dmlc/logging.h

```diff
@@ -251,26 +255,72 @@ class CustomLogMessage {
   std::ostringstream log_stream_;
 };
 
+
+
+#if DMLC_LOG_STACK_TRACE
+inline std::string demangle(char const* msg_str) {
```

Comment:
follow Google C style to use CamelCase for function name

Reply:
ok